{"uid":"476b925205589ac3","name":"test_get_inventDash_details","fullName":"Test_05_InventDashboard.TestInventDashboard#test_get_inventDash_details","historyId":"61f06f8516f15b795953d450f19978a0","time":{"start":1695365890076,"stop":1695365909904,"duration":19828},"description":"Checking Dashboard data","descriptionHtml":"<p>Checking Dashboard data</p>\n","status":"failed","statusMessage":"AssertionError: False != True : Verification Failed to mismatch data.","statusTrace":"self = <TestsCases.Test_05_InventDashboard.TestInventDashboard testMethod=test_get_inventDash_details>\n\n    @allure.description(f\"Checking Dashboard data\")\n    @allure.severity(allure.severity_level.NORMAL)\n    def test_get_inventDash_details(self):\n        self.LoginMethod.nativelogin(\"frugal@latido.com.np\", \"Test@123\")\n        try:\n            self.assertEqual(self.InventDash.execute_info_gather(), True, msg=\"Verification Failed to mismatch data.\")\n        # Checking if assertion failed\n        except (NoSuchElementException, AssertionError, TimeoutException) as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Failed_for_customer\", attachment_type=AttachmentType.PNG)\n>           raise e\n\nTest_05_InventDashboard.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nTest_05_InventDashboard.py:33: in test_get_inventDash_details\n    self.assertEqual(self.InventDash.execute_info_gather(), True, msg=\"Verification Failed to mismatch data.\")\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: in _patched_equals\n    old(self, first, second, msg)\nE   AssertionError: False != True : Verification Failed to mismatch data.","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TestInventDashboard","time":{"start":1695365890075,"stop":1695365890075,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"hasContent":false,"attachmentsCount":0,"shouldDisplayMessage":false}],"testStage":{"description":"Checking Dashboard data","status":"failed","statusMessage":"AssertionError: False != True : Verification Failed to mismatch data.","statusTrace":"self = <TestsCases.Test_05_InventDashboard.TestInventDashboard testMethod=test_get_inventDash_details>\n\n    @allure.description(f\"Checking Dashboard data\")\n    @allure.severity(allure.severity_level.NORMAL)\n    def test_get_inventDash_details(self):\n        self.LoginMethod.nativelogin(\"frugal@latido.com.np\", \"Test@123\")\n        try:\n            self.assertEqual(self.InventDash.execute_info_gather(), True, msg=\"Verification Failed to mismatch data.\")\n        # Checking if assertion failed\n        except (NoSuchElementException, AssertionError, TimeoutException) as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Failed_for_customer\", attachment_type=AttachmentType.PNG)\n>           raise e\n\nTest_05_InventDashboard.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nTest_05_InventDashboard.py:33: in test_get_inventDash_details\n    self.assertEqual(self.InventDash.execute_info_gather(), True, msg=\"Verification Failed to mismatch data.\")\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: in _patched_equals\n    old(self, first, second, msg)\nE   AssertionError: False != True : Verification Failed to mismatch data.","steps":[],"attachments":[{"uid":"5eb782ca46ae1289","name":"Failed_for_customer","source":"5eb782ca46ae1289.png","type":"image/png","size":184822}],"parameters":[],"stepsCount":0,"hasContent":true,"attachmentsCount":1,"shouldDisplayMessage":true},"afterStages":[{"name":"_unittest_setUpClass_fixture_TestInventDashboard::0","time":{"start":1695365955413,"stop":1695365955413,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"hasContent":false,"attachmentsCount":0,"shouldDisplayMessage":false}],"labels":[{"name":"feature","value":"Inventory Dashboard"},{"name":"severity","value":"normal"},{"name":"suite","value":"Test_05_InventDashboard"},{"name":"subSuite","value":"TestInventDashboard"},{"name":"host","value":"0day"},{"name":"thread","value":"9920-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"Test_05_InventDashboard"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":4,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":4},"items":[{"uid":"15145d353cbc422","status":"failed","statusDetails":"AssertionError: False != True : Verification Failed to mismatch data.","time":{"start":1694755371131,"stop":1694755390234,"duration":19103}},{"uid":"8a18a7a0afce8222","status":"failed","statusDetails":"AssertionError: False != True : Verification Failed to mismatch data.","time":{"start":1694155315972,"stop":1694155344633,"duration":28661}},{"uid":"d1b679f98d18b180","status":"failed","statusDetails":"AssertionError: False != True : Verification Failed to mismatch data.","time":{"start":1692940485758,"stop":1692940510391,"duration":24633}}]},"tags":[]},"source":"476b925205589ac3.json","parameterValues":[]}